UnityProject/Assets/Scripts/Health/IElectrocutable.cs

```diff
@@ -0,0 +1,12 @@
+/// <summary>
+/// Implement this on a component to cause that component to do something when electrocuted.
+/// It's up to the component what it does in response to electrocution.
+/// </summary>
+public interface IElectrocutable
```

Comment:
What's the purpose of this interface if only one component is implementing it? Can anything other than players be electrocuted? Even if so, that system isn't currently implemented (current logic only electrocutes LivingHealthBehavior) so we don't have a good idea if the classes involved are designed according to those needs. 

I would suggest adding logic which tests out this / implements IElectrocutable on another component so we can see if there's any issues with the way these classes are designed.